src/covid/pages/lab-results.encounter-list.tsx

```diff
@@ -87,7 +87,14 @@ const columnsLab: EncounterListColumn[] = [
     key: 'labStatus',
     header: 'Status',
     getValue: encounter => {
-      return getObsFromEncounter(encounter, covidTestStatusConcept_UUID);
+      const status = getObsFromEncounter(encounter, covidTestStatusConcept_UUID);
+      const tagColor = status === 'Completed' ? 'green' : status === 'Canclled' ? 'purple' : 'teal';
```

Comment:
@larslemos please use the HEX color codes instead of names and check on zeplin for the right codes. Also the ```const tagColor....``` can be abstracted into a method since it is used in more than one place

Reply:
Also if no ```status``` is defined it should only show ```--``` and not a tag